react/features/deep-linking/components/DeepLinkingMobilePage.web.js

```diff
@@ -129,6 +129,21 @@ class DeepLinkingMobilePage extends Component<Props> {
                     <p className = { `${_SNS}__text` }>
                         { t(`${_TNS}.appNotInstalled`, { app: NATIVE_APP_NAME }) }
                     </p>
+                    <p className = { `${_SNS}__text` }>
+                        { t(`${_TNS}.ifHaveApp`) }
+                    </p>
+                    <a
+                        { ...onOpenLinkProperties }
+                        className = { `${_SNS}__href` }
+                        href = { generateDeepLinkingURL() }
+                        onClick = { this._onOpenApp }>
+                        <button className = { `${_SNS}__button ${_SNS}__button_primary` }>
```

Comment:
@jitsi/jitsi-meet  I am trying to add a button element for a possible PR for https://github.com/jitsi/jitsi-meet/issues/5992, and I am wondering if react modules such as `Button` on `atlaskit/button` ([example](https://github.com/jitsi/jitsi-meet/blob/72137a28114ca19000f2d52e2f6cf8df570b29a0/react/features/settings/components/web/ProfileTab.js#L3)) is encouraged or not. Does it depend on each case?